lib/rubocop/cop/style/fetch_env_var.rb

```diff
@@ -69,9 +69,19 @@ def message_chained_with_dot?(node)
         # - Used as a flag (e.g., `if ENV['X']` or `!ENV['X']`) because
         #   it simply checks whether the variable is set.
         # - Receiving a message with dot syntax, e.g. `ENV['X'].nil?`.
-        # - `ENV['key']` is a receiver of `||=`, e.g. `ENV['X'] ||= y`.
```

Comment:
It looks better to replace the deletion with a simple description of the updated condition.

Reply:
Thank you. I’ll write a brief description.